src/RetryMiddleware.php

```diff
@@ -51,7 +51,7 @@ public function __construct(
      */
     public static function exponentialDelay($retries)
```

Comment:
Could you update the comment here saying what unit we are returning. 

We are returning `ms`